### ISSUE TITLE
Assets resolving in dev mode (1.4.0 release)

### ISSUE THREAD

**Issue by walkman90** [jooby-project/jooby#1122]
It looks like starting from release `1.4.0` there is an issue with resolving font files and images(probably any type of file except styles and scripts) located under `${basedir}/public/`.

In `dev` mode (means `assets.watch` property is enabled by default) `FileSystemAssetHandler` applies and tries to resolve fonts and images from `workdir`, but they are not there, they are under `basedir`.

Is there any additional configuration that I'm missing to fix that or it is a bug?

With `1.3.0` release everything works fine.

**Comment by jknack** [comment]
side effect of #1085

**Comment by jknack** [comment]
Got the bug, but I'm not able to reproduce it :(

Can you add a example app?

Thanks

**Comment by walkman90** [comment]
Sure, here it is https://github.com/walkman90/webpack-starter. 
I forgot to mention that in my project I also use Webpack that's why I have forked the `webpack-starter` project to reproduce the bug, just for the purity of experiment(doubt it can be the cause).

Seems that the root cause is directory naming(`assets`), cause image file under `public/img/` has been resolved ok, but not under `public/assets/images/`.
